static/apps/style/views/left/left-panel.js

```diff
@@ -42,8 +42,8 @@ define(["jquery",
                 this.menu.show(this.sv);
                 this.skins.show(skv);
             },
+
             handleNewMap: function (model) {
```

Comment:
It may be more efficient / readable enforce the rule that every time a new map is selected, this LeftPanel view gets completely replaced (from the style-app file), and all four regions (PanelStylesView, LayerListView, SkinView, and SelectMapView) get initialized at once. To me, this conditional code that renders some things before the model and other things after the model is defined is confusing. It would be clearer (IMO) to enforce the rule that this view is always instantiated with a model. This may also solve the problem of things rendering more than once.